gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -116,7 +116,10 @@ management:
 #    keystore:                    # path to KeyStore (when sslEnabled is true, default null)
 #    keystorePassword:            # KeyStore password (when sslEnabled is true, default null)
 #    keyPassword:                 # password for recovering keys in the KeyStore (when sslEnabled is true, default null)
-
+#    truststore:
+#      path:                      # Path to the truststore (when sslEnabled is true, default null)
+#      type:                      # Type of the truststore, supports jks, pem, pkcs12 (when sslEnabled is true, default null)
+#      password:                  # Truststore password (when sslEnabled is true, default null)
```

Comment:
On one side you have `keystore`, `keystorePassword` and no `type` and for the truststore the syntax is `truststore.path`, `truststore.type`, `truststore.passord`. 

I personally prefer the truststore configuration syntax but it is not aligned with the keystore one. What if we could provide the same kind of syntax for the keystore configuration while keeping the old one for backward compatibility ? That way it will be much more convenient and understandable for new joiners. WDYT ?

Reply:
To be honest, I asked myself the same thing and I totally agree with you.
I was just unsure how to deal with this regarding the doc because making this change is ok (we can make it backward compatible) so the current doc will be still right but if I update the doc then it will contain stuff that is not working with 3.10 😞 .

Otherwise, I need to target 3.10.x 🤔

Reply:
Done, I've done the changes and rebased the branch on 3.10.x 🚀  I will update the doc accordingly when this PR will be ✅